app/jobs/early_years_payments/practitioner_reminder_email_job.rb

```diff
@@ -6,13 +6,23 @@ class PractitionerReminderEmailJob < ApplicationJob
       3 => 4.weeks
     }.freeze
 
+    # Safeguard: Only send reminders for claims submitted by providers within the last 6 months
+    # This prevents historical claims from suddenly receiving emails when deployed to production
+    CUTOFF_DATE = 6.months.ago.freeze
```

Comment:
Minor but this will be set when the class loads rather than when the job runs

```suggestion
    CUTOFF_DATE = 6.months
```
Then use `CUTOFF_DATE.ago`

Probably not an issue in practice as we deploy every day